db/models.py

```diff
@@ -50,3 +57,73 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        related_name="orders"
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        formatted_date = self.created_at.strftime("%Y-%m-%d %H:%M:%S")
+        return formatted_date
```

Comment:
Do not create `formatted_date` variable, return the value directly in the same places in the future